process/block/shardblock_test.go

```diff
@@ -3085,6 +3085,147 @@ func TestShardProcessor_CreateAndProcessCrossMiniBlocksDstMe(t *testing.T) {
 	assert.Equal(t, noOfTxs, uint32(0))
 }
 
+func TestShardProcessor_CreateAndProcessCrossMiniBlocksDstMeShouldErr(t *testing.T) {
+	t.Parallel()
+	haveTimeReturnsBool := func() bool {
+		return true
+	}
+	tdp := mock.NewPoolsHolderFake()
+	txHash := []byte(nil)
+	tdp.Transactions().AddData(txHash, &transaction.Transaction{}, process.ShardCacherIdentifier(1, 0))
+
+	sp, _ := blproc.NewShardProcessor(
+		&mock.ServiceContainerMock{},
+		tdp,
+		&mock.ChainStorerMock{},
+		&mock.HasherStub{},
+		&mock.MarshalizerMock{},
+		&mock.AccountsStub{},
+		mock.NewMultiShardsCoordinatorMock(3),
+		&mock.ForkDetectorMock{},
+		&mock.BlocksTrackerMock{},
+		createGenesisBlocks(mock.NewMultiShardsCoordinatorMock(3)),
+		false,
+		&mock.RequestHandlerMock{},
+		&mock.TransactionCoordinatorMock{},
+		&mock.Uint64ByteSliceConverterMock{},
+	)
+	_, _, _, err := sp.CreateAndProcessCrossMiniBlocksDstMe(3, 2, 2, haveTimeReturnsBool)
+
+	assert.NotNil(t, err)
```

Comment:
Check for a specific error, don't use NotNil

Reply:
done